content/en/technical-advisory-group/architectural-review-committee/ARC-001.en.md

```diff
@@ -7,4 +7,124 @@ draft: false
 lang: "en"
 ---
 
-TODO
+**Date:** March 26th, 2025
+**Location:** [Virtual/In-Person]  
+**Presenter:** William Hearn (Senior Cloud Architect, GC Cloud One)  
+**Attendees:** ARC Members
+**Objective:** Review the Aurora initiative and secure approval for its **strategic direction and technical architecture**.
+
+---
+
+## 1. Executive Summary
+
+### **Context**
+
+The **Aurora Initiative** is a **unified application hosting platform** designed to standardize **managed Kubernetes cloud offerings** across SSC. It integrates **both proprietary (OpenShift) and OSS (Cloud Native Kubernetes) environments**, drawing from operational experience in the **Cloud Native Platform (CNP)**.
+
+### **Why Now?**
+
+- Current departmental cloud-native strategies are **disjointed, costly, and redundant**.
```

Comment:
You could also talk about the move to centralized cloud at SSC, and the new GC Cloud One initiative.